torchaudio/transforms.py

```diff
@@ -798,7 +799,6 @@ def forward(self, specgram: Tensor, mask_value: float = 0.) -> Tensor:
         Returns:
             Tensor: Masked spectrogram of dimensions (..., freq, time).
         """
-
```

Comment:
nit: in terms of cosmetic, you prefer without the space here? :)

Reply:
Yeah. I forgot which language it was about but one of the Google style guides also discourages it for the reason not contributing to readability.